src/watchful.ts

```diff
@@ -51,6 +51,13 @@ export interface WatchfulProps {
    * @default []
    */
   readonly alarmActionArns?: string[];
+
+  /**
+   * If set to true then no cloudwatch dashboards are generated
+   *
+   * @default false
+   */
+  readonly noDashboard?: boolean;
```

Comment:
switch this to a positive. also please move it next to the dashboardName prop just for organizational purposes
```suggestion
   * Whether to generate CloudWatch dashboards
   *
   * @default true
   */
  readonly dashboard?: boolean;
```